pkg/configuration/load_test.go

```diff
@@ -60,7 +59,7 @@ func TestLoadFromConfigMap(t *testing.T) {
 		}
 		cl := test.NewFakeClient(t, createConfigMap("test-config", "toolchain-host-operator", data))
 
-		cl.MockGet = func(ctx context.Context, key client.ObjectKey, obj runtime.Object) error {
+		cl.MockGet = func(ctx context.Context, key client.ObjectKey, obj client.Object) error {
```

Comment:
same here: `runtimeclient` alias for the `sigs.k8s.io/controller-runtime/pkg/client` pkg